core/server/data/xml/rss/index.js

```diff
@@ -91,7 +91,7 @@ generateFeed = function generateFeed(data) {
             htmlContent = processUrls(post.html, data.siteUrl, itemUrl),
             item = {
                 title: post.title,
-                description: post.meta_description || downsize(htmlContent.html(), {words: 50}),
+                description: post.meta_description || post.custom_excerpt || downsize(htmlContent.html(), {words: 50}),
```

Comment:
@rambii thanks for looking at this! One quick change that is required... the `custom_excerpt` should be used if available, then fall back to the others:

`description: post.custom_excerpt || post.meta_description || downsize(htmlContent.html(), {words: 50}),`